lib/Parse/ParseExpr.cpp

```diff
@@ -3451,6 +3465,9 @@ ParserResult<Expr> Parser::parseExprCollection() {
                                     isStartOfDecl() || isStartOfStmt()))
         break;
 
+    if (Tok.isAtStartOfLine())
+        continue;
```

Comment:
Small nit: we use 2 spaces for indentation.

Reply:
Fixed in https://github.com/apple/swift/pull/22714/commits/faecb8d5f15a3421f661bd606158d0394dbb19be .